Cargo.toml

```diff
@@ -15,3 +16,9 @@ miniscript = { version = "1.0.0", features = ["compiler"] }
 rand = "0.7"
 sha2 = "0.9"
 thiserror = "1"
+
+[dev-dependencies]
+bitcoin-harness = { git = "https://github.com/coblox/bitcoin-harness-rs.git", rev = "a695ddc435f5522d7bc35847caf09df85257cc6b" }
```

Comment:
🤔 Wasn't it here already??

Reply:
I believe that was in your draft PR branch. At first it looked like we were gonna do something different but in the end we could have just used your branch.